### PR TITLE
MINOR: Changes for getting automated test running correctly

### PR DESCRIPTION
**Pull request by bbejeck**
### Description

Fixes for the avro console producer and consumer test runner

### CODE REVIEW

**Comment by bbejeck** [comment]
The relevant test passed on this PR, failures unrelated

File: _data/harnesses/console-consumer-producer-avro/kafka.yml [hunk 1]
```diff
@@ -33,8 +33,14 @@ dev:
 
     - title: Create a schema
       content:
-          - action: execute
-            file:
+          - change_directory: console-consumer-producer-avro
```
Comment (bbejeck) [inline _data/harnesses/console-consumer-producer-avro/kafka.yml:36]: This was the fix that was needed to get the test running on semaphore.  The rest of the changes in the PR are what was needed to get the test-harness running in general.

File: _includes/tutorials/console-consumer-primitive-keys-values/kafka/code/tutorial-steps/dev/expected-output-step-two.txt [hunk 2]
```diff
@@ -8,3 +8,4 @@
 4 : -80.832911
 4 : -2.2259418
 7 : 30.838015
+Processed a total of 10 messages
```
Comment (bbejeck) [inline _includes/tutorials/console-consumer-primitive-keys-values/kafka/code/tutorial-steps/dev/expected-output-step-two.txt:11]: This is a side-fix that was causing another test to fail.  I guess it got dropped somehow from one of the rebases.

File: _data/harnesses/console-consumer-producer-avro/kafka.yml [hunk 3]
```diff
@@ -33,8 +33,14 @@ dev:
...
+            action: execute
+            file: tutorial-steps/dev/make-schema-dir.sh
```
Comment (bbejeck) [inline _data/harnesses/console-consumer-producer-avro/kafka.yml:38]: added a step to explicitly create the schema directory. Technically not required, but it fits the model we do with our other KT's

**Comment by bbejeck** [comment]
ping @rspurgeon the tests for the avro-produce-consume are working now

**Review by rspurgeon — APPROVED** [review]
LGTM

**Pull request merged**